src/main/java/mops/model/classes/webclasses/WebApplication.java

```diff
@@ -5,19 +5,24 @@
 import lombok.Builder;
 import lombok.Data;
 import lombok.NoArgsConstructor;
+import mops.model.classes.Module;
+import mops.model.classes.Priority;
+import mops.model.classes.Role;
+
 
 @AllArgsConstructor
 @NoArgsConstructor
 @Data
 @Builder(toBuilder = true)
 public class WebApplication {
-    private String module;
-    private int workload;
-    private int priority;
+    private long id;
```

Comment:
sollte nicht in der WebApplication gerade NICHT die ID drin sein? + es fehlt noch min und maxHour! Wurde das von Marcel noch nicht gepusht?